hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java

```diff
@@ -157,6 +158,11 @@ public String convertToLetter(int size) {
         parameterCreator.addParam("cjv", Integer.toString(clusterClientStats.get(ClientType.JAVA)));
         parameterCreator.addParam("cuptm", Long.toString(clusterUpTime));
         parameterCreator.addParam("nuptm", Long.toString(runtimeMxBean.getUptime()));
+        parameterCreator.addParam("osn", osMxBean.getName());
+        parameterCreator.addParam("osa", osMxBean.getArch());
+        parameterCreator.addParam("osv", osMxBean.getVersion());
+        parameterCreator.addParam("jvmn", runtimeMxBean.getVmName());
+        parameterCreator.addParam("jvmv", System.getProperty("java.version"));
```

Comment:
I think, we should surround this block with a try-catch as most of the `OperatingSystemMxBean` method can throw `SecurityException`. (https://docs.oracle.com/javase/7/docs/api/java/lang/management/OperatingSystemMXBean.html#getName())